tests/test_query_parsing.py

```diff
@@ -40,174 +40,161 @@ async def test_nested_query(self):
+        "correct_fp" :[[]],
+        "correct_service" : [{}],
+        "correct_tables" : [{}],
+        "num_query":1
+    },
+     "test_query_7":{
+        "query_str": '''
+                    SELECT color AS col1, table2.x_min AS col2, table3.frame AS col3
+                    FROM colors02 table1 LEFT JOIN objects00 table2 ON table1.frame = table2.frame
+                    JOIN blobs_00 table3 ON table2.frame = table3.frame;
+                    ''',
+        "normalized_query_str": ("SELECT colors02.color, objects00.x_min, blobs_00.frame "
+                                "FROM colors02 LEFT JOIN objects00 ON colors02.frame = objects00.frame "
+                                "JOIN blobs_00 ON objects00.frame = blobs_00.frame"),
+        "correct_fp" :[[]],
+        "correct_service" : [{}],
+        "correct_tables" : [{}],
+        "num_query":1
+    },
+     "test_query_8":{
+        "query_str": '''
+                    SELECT color, x_min AS col2, colors02.frame AS col3
+                    FROM colors02 JOIN objects00 table2 ON colors02.frame = table2.frame
+                    WHERE color = 'blue' AND x_min > 600;
+                    ''',
+        "normalized_query_str":  ("SELECT colors02.color, objects00.x_min, colors02.frame "
+                                "FROM colors02 JOIN objects00 ON colors02.frame = objects00.frame "
+                                "WHERE colors02.color = 'blue' AND objects00.x_min > 600"),
+        "correct_fp" :[["colors02.color = 'blue'"], ['objects00.x_min > 600']],
+        "correct_service" : [{'colors02'}, {'objects00'}],
+        "correct_tables" : [{'colors02'}, {'objects00'}],
+        "num_query":1
+    },
+     "test_query_9":{
+        "query_str":'''
+                    SELECT color, userfunction(x_min, y_min, x_max, y_max)
+                    FROM colors02 JOIN objects00 table2 ON colors02.frame = table2.frame
+                    WHERE table2.frame > 10000 OR y_max < 800;
+                    ''',
+        "normalized_query_str": ("SELECT colors02.color, userfunction(objects00.x_min, objects00.y_min, "
+                                "objects00.x_max, objects00.y_max) "
+                                "FROM colors02 JOIN objects00 ON colors02.frame = objects00.frame "
+                                "WHERE objects00.frame > 10000 OR objects00.y_max < 800"),
+        "correct_fp" : [['blobs_00.frame > 10000', 'objects00.y_max < 800']],
+        "correct_service" : [{'objects00'}],
+        "correct_tables" : [{'objects00'}],
+        "num_query":1
+    }
+}
+    for i in range(1,10):
+      self._test_query(queries["test_query_"+str(i)]["query_str"], config, queries["test_query_"+str(i)]["normalized_query_str"], queries["test_query_"+str(i)]["correct_fp"],queries["test_query_"+str(i)]["correct_service"], queries["test_query_"+str(i)]["correct_tables"], queries["test_query_"+str(i)]["num_query"])
```

Comment:
refactor '_test_query' to receive only two parameters "queries[f'test_query_{i}']" and config